layouts/partials/modules/products.css

```diff
@@ -68,6 +68,19 @@
   height: 100%;
 }
 
+.product-list li .color-dots > a.color-dot--see-more {
+  background-color: white;
+  color: black;
+  font-size: 1.5rem;
+  line-height: 1rem;
+  text-decoration: none;
+}
+
+.product-list li .color-dots > a.color-dot--see-more:hover {
+  background-color: #BA001F;
+  color: #d2092d;
```

Comment:
The `color` here is available as the custom property `var(--color-reima-red)`. The background color should probably be a custom property, but is not. Can you create this as a custom property in `baseof.css` with some good name?

Also, this becomes a weird red-on-red button on hover. I think the text `color` should be white on hover, as this is most likely a bug in the design prototype.

Reply:
I adden new custom property --color-cta-hover for #BA001F  (Also find and replaced this property to few similar styles)
That red-on-red was copied from proto, white color looks better so I changed it.